main/src/com/google/refine/model/medadata/validator/checks/MaximumConstraint.java

```diff
@@ -26,9 +26,9 @@ public boolean checkCell(Cell cell) {
         boolean valid = true;
         
         try {
-            Comparable value = field.castValue(cell.value.toString(), false);
+            Comparable value = column.castValue(cell.value.toString());
             // return this - threashold
-            if (value.compareTo(field.castValue(threashold, false)) > 0)
+            if (value.compareTo(column.castValue(threashold)) > 0)
```

Comment:
there is a typo here: `threashold` should be `threshold` (and at other places)

Reply:
fixed